### PR TITLE
Launch Portland 2025 CFP

### PR DESCRIPTION
**Pull request by ericholscher**
This is just a draft PR to do some initial testing,
but will eventually get content updates before we go live.

----
📚 Documentation preview 📚: https://writethedocs-www--2241.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/portland/2025/cfp.rst
Comment (smoreface) [inline docs/conf/portland/2025/cfp.rst]:
```suggestion
How does your talk help improve the quality of documentation everywhere? Are you trying a new approach? Are you solving a known problem? Have you done the research into how folks have tried to solve your issue before?
```

File: docs/conf/portland/2025/cfp.rst
Comment (smoreface) [inline docs/conf/portland/2025/cfp.rst]: Love this edit!

**Comment by ericholscher** [comment]
@plaindocs We aren't necessarily gonna ship the whole site w/ the CFP if we haven't reviewed it all, but we can always just swap the full site flag before shipping if needed.

File: docs/conf/portland/2025/cfp.rst
Comment (hzoppetti) [inline docs/conf/portland/2025/cfp.rst]: We'll ask you for your pronouns, how you pronounce your name, and some interesting facts about you so our emcee can prepare to introduce you.

Comment (plaindocs) [inline docs/conf/portland/2025/cfp.rst]:
```suggestion
   We'll ask you for your pronouns, how you pronounce your name, and some interesting facts about you so our emcee can prepare to introduce you.
```

**Comment by plaindocs** [comment]
> @plaindocs We aren't necessarily gonna ship the whole site w/ the CFP if we haven't reviewed it all, but we can always just swap the full site flag before shipping if needed.

Ah yeah, soz for coopting your PR in that case :-)

File: docs/conf/portland/2025/cfp.rst
Comment (plaindocs) [inline docs/conf/portland/2025/cfp.rst]:
```suggestion
We *strongly* prefer talks presented by one *single speaker*. If you're planning to submit with a co-speaker, please let us know why, and take note that it will affect the chances of your talk being selected.
```

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (ericholscher) [inline docs/conf/portland/2025/news/announcing-cfp.rst]:
```suggestion
We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, opening even earlier than last year, even though the conference is back at the start of May this year.
```

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (ericholscher) [inline docs/conf/portland/2025/news/announcing-cfp.rst]:
```suggestion
We're also launching our `sponsorship prospectus <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/sponsors/prospectus/>`_ with all the information for sponsoring the conference.
```

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (ericholscher) [inline docs/conf/portland/2025/news/announcing-cfp.rst]:
```suggestion
Our `sponsorship prospectus <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/sponsors/prospectus/>`_ is updated for {{ year }},
```

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (ericholscher) [inline docs/conf/portland/2025/news/announcing-cfp.rst]:
```suggestion
We're focusing on in-person interaction at the conference, taking full advantage of {{ about.venue }} with a dedicated space for sponsors in the hallway outside the main venue.
```

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (ericholscher) [inline docs/conf/portland/2025/news/announcing-cfp.rst]:
```suggestion
As always, we try to keep ticket prices low so that the event is accessible to the widest range of people.
If you can't afford the tickets and still wish to attend, we will launch our grants program soon after opening ticket sales.
```

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (ericholscher) [inline docs/conf/portland/2025/news/announcing-cfp.rst]: I might drop this, yea? We don't have anything firm here, so probably not worth mentioning at this stage.

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (ericholscher) [inline docs/conf/portland/2025/news/announcing-cfp.rst]:
```suggestion
Sign up to one or more of our `mailing lists <http://eepurl.com/cdWqc5>`_. Your information will never be shared with any third parties, and you can unsubscribe at any time.
```

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (ericholscher) [inline docs/conf/portland/2025/news/announcing-cfp.rst]:
```suggestion
Want to connect with other documentarians in real-time? Join our `Slack <https://www.writethedocs.org/slack/>`_.
```

**Review by ericholscher — COMMENTED** [review]
Copy edited the mailing -- overall looks solid.

File: docs/conf/portland/2025/cfp.rst
Comment (diggerlilly) [inline docs/conf/portland/2025/cfp.rst]:
```suggestion
Note that all speakers must read, understand, and agree to our :doc:`/code-of-conduct`. All talks and slides will need to follow our Code of Conduct. If you are unsure about any aspect of this, please ask us for clarification.
```

File: docs/conf/portland/2025/cfp.rst
Comment (diggerlilly) [inline docs/conf/portland/2025/cfp.rst]:
```suggestion
    We'll let you know whether your proposal was accepted, and ask for some supplementary information about you.
```

File: docs/conf/portland/2025/cfp.rst
Comment (diggerlilly) [inline docs/conf/portland/2025/cfp.rst]:
```suggestion
    We'll ask for a copy of your slides so our human captioners can prepare for your talk
```

File: docs/conf/portland/2025/cfp.rst
Comment (diggerlilly) [inline docs/conf/portland/2025/cfp.rst]:
```suggestion
If you have any questions about any part of this process, please email us at `{{email}} <mailto:{{email}}>`_; we'll be happy to chat.
```

File: docs/conf/portland/2025/cfp.rst
Comment (diggerlilly) [inline docs/conf/portland/2025/cfp.rst]: I feel like the first letter of these words should be capitalized since they're basically full sentences preceded by a bolded category.

so here, s/it's/It's  and so on

File: docs/conf/portland/2025/cfp.rst
Comment (diggerlilly) [inline docs/conf/portland/2025/cfp.rst]:
```suggestion
- **Research** We don't need all talks to be about an entirely new topic, but if you're suggesting a talk that looks really similar to one that was given last year, demonstrate that you realize this and mention why yours is different.
```

also, maybe s/last year/in recent years

File: docs/conf/portland/2025/cfp.rst
Comment (diggerlilly) [inline docs/conf/portland/2025/cfp.rst]:
```suggestion
- **Example proposal** Check out our example so you know what we expect to see in each field.
```
or something along these lines, for parallel construction

File: docs/conf/portland/2025/cfp.rst
Comment (diggerlilly) [inline docs/conf/portland/2025/cfp.rst]: include date?

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (diggerlilly) [reply docs/conf/portland/2025/news/announcing-cfp.rst]: There are lots of words here that are tripping me up with possible double meanings ( `conference is back at the start...` etc). Also, I'm not sure folks will notice/fully appreciate the relative date shifts (while acknowledging here that it is notable :) ). Does the wider delta give submitters additional time to work on proposals or to polish them once approved?

In any case, just suggestions:

```suggestion
We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, which is opening earlier this year than last year, giving you __[more time to blah blah]__ since the conference is in May instead of April this year.
```
or just:
We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, which is opening earlier this year!

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (diggerlilly) [inline docs/conf/portland/2025/news/announcing-cfp.rst]:
```suggestion
If there's something you'd really like to see a talk on this year, submit a proposal, or send the submission link to someone else you would like to hear from!
```
(how does "refer" work?)

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (diggerlilly) [reply docs/conf/portland/2025/news/announcing-cfp.rst]: s/focussing/focusing

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (diggerlilly) [inline docs/conf/portland/2025/news/announcing-cfp.rst]:
```suggestion
We hope you will join us in person for yet another Write the Docs conference.
```

**Review by diggerlilly — APPROVED** [review]
Some edits and suggestions

File: docs/conf/portland/2025/cfp.rst
Comment (plaindocs) [reply docs/conf/portland/2025/cfp.rst]: Is this a US / UK thing I wonder?

File: docs/conf/portland/2025/cfp.rst
Comment (plaindocs) [inline docs/conf/portland/2025/cfp.rst]:
```suggestion
You'll be able to edit your proposal up until the submission deadline. Please be considerate of our reviewers when making changes to talks you've already submitted.
```

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (plaindocs) [inline docs/conf/portland/2025/news/announcing-cfp.rst]:
```suggestion
We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, opening even earlier than last year.
```

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (plaindocs) [inline docs/conf/portland/2025/news/announcing-cfp.rst]:
```suggestion
```

File: docs/conf/portland/2025/cfp.rst
Comment (plaindocs) [inline docs/conf/portland/2025/cfp.rst]:
```suggestion
- **Spoilers** It's pretty normal not to want to include your main point in your abstract, but please make sure to highlight it for the selection committee!
- **Research** We don't need all talks to be about an entirely new topic, but if you're suggesting a talk that looks really similar to one that was given last year, demonstrate that you realize this and mention why yours is different.
- **Tooling** We're pretty serious about preferring talks about people, process or principles than talks about tooling. If you are submitting a proposal about tooling, tell us what makes this one special.
- **Example proposal** Check out our example so you know what we expect to see in each field.
```

File: docs/conf/portland/2025/cfp.rst
Comment (plaindocs) [reply docs/conf/portland/2025/cfp.rst]: If I have to :-p. It should for sure at least be consistent.

**Comment by plaindocs** [comment]
Thanks for the eyes folks, I've merged all of that, with a couple of minor changes.

File: docs/_data/portland-2025-config.yaml
Comment (ericholscher) [inline docs/_data/portland-2025-config.yaml]: We need info much before that for social media posting, etc.

Comment (plaindocs) [reply docs/_data/portland-2025-config.yaml]: Yep, this is reverted in the CFP page, and I'll remove from here as well.

Comment (plaindocs) [reply docs/_data/portland-2025-config.yaml]: I also need to update the emails to go hand in hand, but that doesn't need to block this PR.

**Comment by plaindocs** [comment]
@ericholscher made some minor updates, will update the emails page separately. Wanna merge and post on Monday?

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (ericholscher) [inline docs/conf/portland/2025/news/announcing-cfp.rst:45]: We already mentioned this in the opening, so probably worth including again.

```suggestion
Tickets will go on sale in {{ date.tickets_live }}.

Prices are the same as last year:

* **Student or Unemployed Tickets**: *{{tickets.student.price}}*
* **Independent Tickets**: *{{tickets.independent.price}}*
* **Corporate Tickets**: *{{tickets.corporate.price}}*
```

**Review by ericholscher — COMMENTED** [review]
Looks good to ship.

**Comment by katiejano** [comment]
What date do you want to announce speakers? 

Right now, we have CFPs closing on January 21 and February 4 to notify speakers of whether they're speaking. We'll need at least a week to get confirmation from everyone that they're still able to speak. Are you wanting to announce February 12 or 13? That's a pretty quick turnaround so I am wondering if we should move up a date earlier.

**Comment by katiejano** [comment]
Edits to the CFP page: 

Under Conference goals
- Is the last paragraph supposed to be larger font and italicized? 

Under Speaker benefits & logistics
- Benefits - need to bullet point out list
- Logistics - delete last paragraph as it’s a duplicate starting with, “Note that all speakers must read…”

Under Submit your proposal
- Move the following bullet points to a new section as this information needs to come earlier, “Spoilers, Research, Tooling, Example Proposal”. I called this Tips and added it [hackmd](https://hackmd.io/zjV2eVpaTO-xwMYq7Obz3w?both)

**Comment by plaindocs** [comment]
> Right now, we have CFPs closing on January 21 and February 4 to notify speakers of whether they're speaking. We'll need at least a week to get confirmation from everyone that they're still able to speak. Are you wanting to announce February 12 or 13? That's a pretty quick turnaround so I am wondering if we should move up a date earlier.

That's for the private confirmations, the announcement can come about a week after that yeah. But we don't need to publicize that either. So no action needed.

I've made the other edits, good calls there.

File: docs/conf/portland/2025/news/announcing-cfp.rst
Comment (plaindocs) [inline docs/conf/portland/2025/news/announcing-cfp.rst:45]:
```suggestion
Tickets will go on sale in {{ date.tickets_live }}.

Prices are the same as last year:

* **Student or Unemployed Tickets**: *{{tickets.student.price}}*
* **Independent Tickets**: *{{tickets.independent.price}}*
* **Corporate Tickets**: *{{tickets.corporate.price}}*
```

**Pull request merged**
